utils/helper.sh

```diff
@@ -52,8 +52,8 @@ function validate_version_confluent_cli_for_cp() {
 
   validate_version_confluent_cli_v2 || exit 1
 
-  VER_MIN="1.11.0"
-  VER_MAX="1.22.0"
+  VER_MIN="1.26.0"
+  VER_MAX="1.26.1"
```

Comment:
Since we don't know max at this time, this hedges a bit.

Reply:
There is definitely a 1.27.0. I was trying to think of a clever way to automate updating this based on the results from `curl -sL --http1.1 https://cnfl.io/cli | sh -s -- -l`, but this seems like maybe more trouble than it's worth.

Reply:
We explicitly do NOT want to curl because when 7.0 is released with a new and possibly incompatible CLI, 6.2.x should NOT use it.